web/src/components/njwds-extended/ButtonDropdown.tsx

```diff
@@ -56,7 +56,7 @@ export const ButtonDropdown = (props: Props): ReactElement => {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const DefaultButton = (props: { children: ReactNode; [key: string]: any }): ReactElement => (
-    <UnStyledButton style="tertiary" textBold smallText {...props} />
+    <UnStyledButton style="default" isTextBold isSmallerText {...props} />
```

Comment:
[Dust] I think hasSmallerText would be more gramatically apealing to me

Reply:
We previously ran into instances where the developers had a hard time figuring out what properties were available, so this "is-" prefix was added. It allows us to quickly discern what options are available using some of the tools that are built into VS code.